rasa/core/processor.py

```diff
@@ -150,7 +206,10 @@ def predict_next(self, sender_id: Text) -> Optional[Dict[Text, Any]]:
             message.text = self.message_preprocessor(message.text)
         # we have a Tracker instance for each user
         # which maintains conversation state
-        tracker = self._get_tracker(message.sender_id)
+        tracker = await self.get_tracker_with_session_start(
```

Comment:
should we not change all `_get_tracker` occurrences to `get_tracker_with_session_start` ?